test/test_data/birk5.ine

```diff
@@ -0,0 +1,31 @@
+birk5.ine
```

Comment:
all these files are duplicates with files in ```data```

Reply:
Yes everything in folder notincluded it is duplicated and not included. I am going to remove this folder.